src/options/options.html

```diff
@@ -107,6 +107,16 @@
 						</ul>
 					</fieldset>
 				</li>
+				<li>
+					<label data-i18n="__MSG_optionQrQuietZone__" for="qrQuietZone">QR quiet zone: </label>
+					<input class="setting save-on-change trigger-on-update" type="range" value="0" min="0" max="10" step="1" id="qrQuietZone" name="qrQuietZone">
```

Comment:
```suggestion
					<input class="setting save-on-change trigger-on-update" type="range" value="0" min="0" max="5" step="1" id="qrQuietZone" name="qrQuietZone">
```

I guess 10 as a padding is a little extreme. Should not 5 satisfy? Or do you really think someone is going to need more?